internal/session/sessionwants.go

```diff
@@ -12,7 +11,6 @@ import (
 // sessionWants keeps track of which cids are waiting to be sent out, and which
 // peers are "live" - ie, we've sent a request but haven't received a block yet
 type sessionWants struct {
```

Comment:
The changes in sessionWants is basically just removing a bunch of locks that we no longer need as everything is inside the Session run loop now